app/models/theme.rb

```diff
@@ -59,6 +59,7 @@ def theme_modifier_set
     end
 
     Theme.expire_site_cache! if saved_change_to_user_selectable? || saved_change_to_name?
+    Theme.expire_site_cache! if saved_change_to_color_scheme_id?
```

Comment:
why don't we add this condition to line 61 ?

Reply:
It read better to me as a separate line, but I don't have strong feelings about it, made the suggested change.